docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1198,6 +1198,7 @@ Currently, the following Quarkus extensions support this feature:
 * quarkus-jdbc-mssql
 * quarkus-jdbc-mysql
 * quarkus-jdbc-postgresql
+* quarkus-kafka-client
```

Comment:
You may add the kafka connector from reactive messaging too (even if it's by transitivity)